docs/demo_api.rst

```diff
@@ -215,21 +215,6 @@ JavaScript API
          :throws SomeError: For whatever reason in that case.
          :returns: Something.
 
-Generated Index
-===============
-
-.. rst-example::
-
-   A generated index (:ref:`genindex`) is part of the Sphinx build process, unless
-   `html_use_index` is set to `False`.
-
-   Sphinx also allows indexing by domain (programming language), as seen in the
-   :ref:`modindex` for the demo Python module that is documented on this page.
-
-.. note::
-   This theme does not support a separate search page (usually referenced with
-   ``:ref:`search``), since the search is accessible in the site's navigation bar.
-
```

Comment:
I guess it is still valid.  I reverted this change.

However, the reason I removed it is because this theme doesn't link to those pages by default, and they are redundant with search so it seems like there isn't much point in advertising them.  In the tensorstore docs I disable the index generation.

Reply:
I only asked because `sphinx-quickstart` automatically adds a section with those references to _index.rst_.